packages/cli/src/commands/versions/bump.ts

```diff
@@ -255,7 +255,13 @@ export default async (opts: OptionValues) => {
       );
     }
 
-    await runYarnInstall();
+    if (opts.skipInstall === undefined) {
```

Comment:
Very minor thing but I think we could be simplified a little bit with
```suggestion
    if (!opts.skipInstall) {
```

Looks like that's what we do in [createApp](https://github.com/backstage/backstage/blob/master/packages/create-app/src/createApp.ts#L121)